Ruby/Ruby.sublime-syntax

```diff
@@ -68,7 +68,7 @@ file_extensions:
   - thor
   - Thorfile
   - Vagrantfile
-first_line_match: ^#!/.*\bruby
+first_line_match: ^#!\s*/.*\bruby\b
```

Comment:
I've seen `erb` in shebang lines.  Also `jruby`.  But both of those are vanishingly rare.